package.json

```diff
@@ -12,7 +12,9 @@
     "pretty": "prettier --write --print-width 80 --single-quote --trailing-comma es5 --tab-width 2 'src/**/*.js' 'tests/**/*.js'",
     "create-migration": "knex migrate:make",
     "migrate": "knex migrate:latest",
-    "migrate-rollback": "knex migrate:rollback"
+    "migrate-rollback": "knex migrate:rollback",
+    "docker:migrate": "docker exec pantry-list-api_dev_1 ./node_modules/.bin/knex migrate:latest",
+    "docker:migrate-rollback": "docker exec pantry-list-api_dev_1 ./node_modules/.bin/knex migrate:rollback"
```

Comment:
I don't _love_ that the container name is hardcoded here but it was easy and seemed like it would be fairly safe given our use case.